kola/tests/misc/network.go

```diff
@@ -127,6 +127,7 @@ func NetworkListeners(c cluster.TestCluster) {
 		{"tcp", "22", "systemd"},          // ssh
 		{"udp", "68", "systemd-network"},  // dhcp6-client
 		{"udp", "546", "systemd-network"}, // bootpc
+		{"tcp", "10010", "containerd"},    // containerd
```

Comment:
https://github.com/moby/moby/issues/37507#issuecomment-406654334

It references `disabled_plugins = ["cri"]` in the containerd configuration file, that was changed for the Edge channel. I wonder if that was a correct change: shouldn't cri be using a unix socket instead of a tcp socket?

Reply:
I don't know if it works with a unix socket.
I'm fine with dropping the containerd change from this PR, until we could test it again with a new containerd config.